src/main/java/net/masterthought/cucumber/Configuration.java

```diff
@@ -136,13 +138,31 @@ public String getProjectName() {
         return projectName;
     }
 
+    /**
+     * Sets directory qualifier.
+     *
+     * @param directoryQualifier directory qualifier
+     */
+    public void setDirectoryQualifier(String directoryQualifier) {
```

Comment:
was it intentional to use similar name to https://github.com/damianszczepanik/cucumber-reporting/commit/3b3bef7a8d32841310587629e607bab7c1a9af75#diff-8bf6b44089f1dce48374088686c23f67R78 ?

Reply:
Nope, just happened to be this.  I'm actually thinking about changing it to `directorySuffix` - let me know if it's preferred.

Reply:
Then recommend to change it to avoid confusing